website/src/pages/demodays.js

```diff
@@ -11,20 +12,18 @@ const Hello = () => {
             <div>
                 <section className={styles.intro}>
                     <div>
-                    <h1 className={styles.title}>Demo Days 2022</h1>
-                    <h2 className={styles.date}> 🗓️ 22<sup>nd</sup> October</h2>
+                        <h1 className={styles.title}>Demo Days 2022</h1>
+                        <h2 className={styles.date}> 🗓️ 22<sup>nd</sup> October</h2>
                     </div>
                     <p className={styles.sub}>
-                    On October 22 the final videos and demonstrations from all the 3 projects of GSoC 2022 will be shown.
+                        On October 22 the final videos and demonstrations from all the 3 projects of GSoC 2022 will be shown.
                     </p>
                 </section>
                 <section className={styles.about}>
                     <div className={styles.descript}>
                         <h1 className={styles.head}>Agenda</h1>
                         <p className={styles.headscript}>
-                        6:00 PM(IST)
-                        GSoC presentations
-On October 22 the final videos and demonstrations from 3 projects of GSoC 2022 will be shown. 
+                            6:00 PM(IST),GSoC presentations On October 22.The final videos and demonstrations from 3 projects of GSoC 2022 will be shown.
```

Comment:
This can be changed to :

Agenda 

On October 22, 12:30 PM GMT, we will have our GSoC mentees showcasing their work and learnings to the community

Reply:
Okay